docs/examples.rst

```diff
@@ -610,7 +610,7 @@ If you'd like to enforce it, ``attrs`` will try to help:
    ... class C(object):
    ...     x = attr.ib()
    >>> i = C(1)
-   >>> i.x = 2
+   >>> i.x = 2  # type: ignore
```

Comment:
What error are you seeing here?

Reply:
The expected error about the read-only attribute. I’ll clarify that with a comment.

Reply:
Oh frozen. Missed that.  Cool!